db/migrate/20170316160949_add_auth_bypass_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthBypassId < ActiveRecord::Migration[5.0]
+  def change
+    add_column :access_limits, :auth_bypass_ids, :json, null: false, default: []
```

Comment:
@kevindew when running the migration locally, it's quick: it takes 0.299s. 
For the fact_check_ids column, we'll only need to keep it for a few hours, until we've republished all draft content so they all have an auth_bypass_id.